code/datums/brain_damage/special.dm

```diff
@@ -245,14 +245,15 @@
 
 /datum/brain_trauma/special/psychotic_brawling/on_gain()
 	..()
-	psychotic_brawling = new(null)
+	psychotic_brawling = new()
```

Comment:
new... what?
Please don't use implict new on var sets if the var is not a typed local var

Reply:
wat? it's literally three lines above